### PR TITLE
Feature ➤ Simple CameraX

### PR DESCRIPTION
**Pull request by EAlexisDuque**
## Description
Create a new class that allows you to use the new cameraX library with the design for the wallet application

## Trello Cards
* [Migrate to camera X [8D]](https://trello.com/c/H7RtwB7t/1281-migrate-to-camera-x-8d)

### CODE REVIEW

**Comment by EAlexisDuque** [comment]
## UI Test
Screen Shot 2019-11-22 at 3 39 24 PM

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -57,6 +57,6 @@
     <color name="emerald_disable_file_selector">#C9C9C9</color>
     <color name="emerald_enable_file_selector_icon">#5542A5F5</color>
 
-
-
+    <!-- WALLET CAMERA -->
```
Comment (cparracondor) [inline app/src/main/res/values/colors.xml]: This camera it's not only for Wallet

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -63,4 +63,12 @@
     <dimen name="skeleton_title_margin_top">40dp</dimen>
     <dimen name="content_skeleton_margin_top">86dp</dimen>
     <dimen name="bottom_content_skeleton_margin_bottom">32dp</dimen>
+
+    <!-- WALLET CAMERA -->
```
Comment (cparracondor) [inline app/src/main/res/values/dimens.xml]: Same

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -49,4 +58,50 @@
         <item name="android:windowFullscreen">true</item>
         <item name="android:windowIsFloating">false</item>
     </style>
+
+    <!-- WALLET CAMERA -->
```
Comment (cparracondor) [inline app/src/main/res/values/styles.xml]: Same

Comment (EAlexisDuque) [reply app/src/main/res/values/styles.xml]: Done

File: app/src/main/res/values/dimens.xml (hunk 2 above)
Comment (EAlexisDuque) [reply app/src/main/res/values/dimens.xml]: Done

**Pull request merged**